src/content/en/tools/workbox/guides/advanced-recipes.md

```diff
@@ -15,32 +15,42 @@ worker has updated and waiting to install.
 
 To do this you'll need to add some code to your page and to your service worker.
 
-**Add to your page**
+**Code in your page**
 
 ```html
 <script type="module">
-import {Workbox} from 'https://storage.googleapis.com/workbox-cdn/releases/4.0.0/workbox-window.prod.mjs';
+import {Workbox} from 'https://storage.googleapis.com/workbox-cdn/releases/4.1.0/workbox-window.prod.mjs';
```

Comment:
Can we use the import that automatically adds the latest version?

BTW, I think in order to use it you have to use `<pre class="prettyprint">`...

Reply:
I'll check that out.

Reply:
So https://github.com/google/WebFundamentals/blob/master/src/content/en/tools/workbox/_shared/workbox-sw-cdn-url.html explicitly includes the full URL to `workbox-sw.js`

We can update that file to just have the URL prefix, and then update the other docs that use the include file to combine the prefix + `workbox-sw.js`, and then update this usage to be the prefix + `workbox-window.mjs`.

How about doing that in another PR, though?

Reply:
Oh, right, this is a reference to `workbox-window` ... yeah, let's do that in another PR.